docs/support/index.rst

```diff
@@ -122,7 +122,7 @@ ____________________________________
 * :code:`dest_file_system` (ios) - Destination file system for SCP transfers (default: ``flash:``).
 * :code:`auto_rollback_on_error` (ios) - Disable automatic rollback (certain versions of IOS support configure replace, but not rollback on error) (default: ``True``).
 * :code:`global_delay_factor` (ios, vyos) - Allow for additional delay in command execution (default: ``1``).
-* :code:`nxos_protocol` (nxos) - Protocol to connect with.  Only 'https' and 'http' allowed (default: ``http``).
+* :code:`transport` (eos, ios, nxos) - Protocol to connect with. Allowed transports depends on the underlying library, but typically ``http``/``https`` and/or ``telnet``/``ssh``. Defaults to ``https`` on EOS and NXOS and ``ssh`` on IOS.
```

Comment:
I suggest having very specific options, perhaps a sub-list for each driver so it's clear rather than referencing other documents of the underlying library, i.e.:

- :code:`transport` (eos, ios, nxos) - Protocol to connect with. Allowed transports depends on the underlying library
  - ios: telnet, ssh. Default: ssh
  - eos: http, https. Default: https
  - nxos: http, https. Default: http

etc. Makes sense? Anyway, this will be delayed till we release the versions containing these changes.